cogs/gamerwords.py

```diff
@@ -70,42 +71,51 @@
 	CATCHPHRASES = list(map(str.rstrip, f))
 
 
+class Break(Exception):
+	pass
+
+
 class UnicodeAwareBruhReplacer:
```

Comment:
of course it's unicode aware. please rename to GamerReplacer for consistency  with the rest of the codebase as well.